docs(en-US): references -> user profile -> identities (#99)
logto-io/docs: docs/docs/references/users/README.md

:::

### Identities

> _Identities_ contain the user info retrieved from social sign-in

Author: ```suggestion
_Identities_ contains the user info retrieved from social sign-in
```

> ### Identities
> 
> _Identities_ contains the user info retrieved from social sign-in

Reviewer: See discussion:
https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1656647734811709

Maybe:
```suggestion
### identities

_identities_ contains the user info retrieved from social sign-in
```

(i.e., sign-in with a social connector).

The user info varies by provider,
but it will typically include the following:

> - Name of this provider, such as lowercase "facebook", "google", or "wechat"

Author: ```suggestion
- [Target](#TODO-link-to-connector-reference) of this provider, such as lowercase "facebook", "google", or "wechat"
```

- User's unique identifier for this provider
- User's name
- User's verified email
- User's avatar

> The user may link their account to multiple social identity providers via social sign-in, such as Facebook, Google, and Wechat;

Author: ```suggestion
The user's account may be linked to multiple social identity providers via social sign-in, such as Facebook, Google, and WeChat;
```

:::

### Identities

_Identities_ contain the user info retrieved from social sign-in

> (a.k.a. sign-in with a social connector).

Reviewer 2: i.e.

Reviewer: should be `contains` as you used _Identites_ as a single subject?

Author: updated

Author: updated `contains`

> (a.k.a. sign-in with a social connector).

Author: ```suggestion
(i.e., sign-in with a social connector).
```

> The user info from individual social identity providers varies by provider,

Reviewer 2: The user info varies by the provider

Author: updated.

> The user info from individual social identity providers varies by provider,

Author: ```suggestion
The user info varies by provider,
```

> but it will typically include the following:

Reviewer 2: Sounds strange

Reviewer 3: OK to me

Author: According to another doc
<img width="236" alt="image" src="https://user-images.githubusercontent.com/10594507/176816787-0e093f7d-aa15-4369-91d3-d8a79d1f5f33.png">

update to?

```
and it typically includes the following:
```

> - Name of this provider, such as lowercase "facebook", "google", or "wechat"

Reviewer: `Name` -> `Target` to align with the term in connector. Also, add a link to the "connector reference" page so that developers can explore more details.

Author: Updated.

- User's unique identifier for this provider
- User's name
- User's verified email
- User's avatar

> The user may link their account to multiple social identity providers via social sign-in, such as Facebook, Google, and Wechat;

Reviewer: use `WeChat` to express the meaning of "wechat" across all UI and docs.

Author: Fixed

> The user may link their account to multiple social identity providers via social sign-in, such as Facebook, Google, and Wechat;

Reviewer 2: their?

Author: Improve by Grammarly
<img width="925" alt="image" src="https://user-images.githubusercontent.com/10594507/176816695-ebf1de81-eb40-4980-a313-8af266f4d0e4.png">

Reviewer 3: lol, but it's obvious wrong

Author: Updated to 

```
The user's account may be linked to multiple social identity providers via social sign-in
```

the related user info retrieved from these providers will be stored in the _identities_ object.

Sample _identities_ from a user:

```json
{
  "facebook": {
    "userId": "5110888888888888",
    "details": {
      "id": "5110888888888888",
      "name": "John Joe",
      "email": "johnjoe@logto.io",
      "avatar": "https://logto.dev/console/avatar-006.89b02841.png"
    }
  },
  "github": {
    "userId": "10599999",
    "details": {
      "id": "10599999",
      "name": "John Joe",
      "email": "johnjoe@logto.io",
      "avatar": "https://logto.dev/console/avatar-004.bbfb95ff.png"
    }
  },
  "google": {
    "userId": "100001553333333333333",
    "details": {
      "id": "100001553333333333333",
      "name": "John Joe",
      "email": "johnjoe@logto.io",
      "avatar": "https://logto.dev/console/avatar-007.e3c47716.png"
    }
  },
  "wechat": {
    "userId": "O8sU-6JWMMNZzuXo6-xaEjouyQZ8",
    "details": {
      "id": "O8sU-6JWMMNZzuXo6-xaEjouyQZ8",
      "name": "John Joe",
      "avatar": "https://logto.dev/console/avatar-005.52916e5f.png"
    }
  }
}

> ```

Reviewer: I would say 2 is enough to show that you can have multiple _identities_s

Author: updated

> ```json
> {
>   "facebook": {
>     "userId": "5110888888888888",
>     "details": {
>       "id": "5110888888888888",
>       "name": "John Joe",
>       "email": "johnjoe@logto.io",
>       "avatar": "https://logto.dev/console/avatar-006.89b02841.png"
>     }
>   },
>   "github": {
>     "userId": "10599999",
>     "details": {
>       "id": "10599999",
>       "name": "John Joe",
>       "email": "johnjoe@logto.io",
>       "avatar": "https://logto.dev/console/avatar-004.bbfb95ff.png"
>     }
>   },
>   "google": {
>     "userId": "100001553333333333333",
>     "details": {
>       "id": "100001553333333333333",
>       "name": "John Joe",
>       "email": "johnjoe@logto.io",
>       "avatar": "https://logto.dev/console/avatar-007.e3c47716.png"
>     }
>   },
>   "wechat": {
>     "userId": "O8sU-6JWMMNZzuXo6-xaEjouyQZ8",
>     "details": {
>       "id": "O8sU-6JWMMNZzuXo6-xaEjouyQZ8",
>       "name": "John Joe",
>       "avatar": "https://logto.dev/console/avatar-005.52916e5f.png"
>     }
>   }
> }
> ```

Author: ```suggestion
```json
{
  "facebook": {
    "userId": "5110888888888888",
    "details": {
      "id": "5110888888888888",
      "name": "John Joe",
      "email": "johnjoe@logto.io",
      "avatar": "https://logto.dev/console/avatar-006.89b02841.png"
    }
  },
  "wechat": {
    "userId": "O8sU-6JWMMNZzuXo6-xaEjouyQZ8",
    "details": {
      "id": "O8sU-6JWMMNZzuXo6-xaEjouyQZ8",
      "name": "John Joe",
      "avatar": "https://logto.dev/console/avatar-004.bbfb95ff.png"
    }
  }
}
```
```
